### PR TITLE
LinkInstance wrappers and tests

### PR DESCRIPTION
**Pull request by dariaivanciucova**
### Purpose

This PR adds wrappers for the RevitLinkInstance class.
Screenshot 2023-07-27 162803

- Renamed LinkInstance to RevitLinkInstance 
- Added Revit Nodes tests for RevitLinkInstance and LinkElement methods

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] Snapshot of UI changes, if any.

### Reviewers

@Mikhinja @reddyashish 

### FYIs

@dnenov @Amoursol

### CODE REVIEW

**Review by Mikhinja — APPROVED** [review]

File: test/Libraries/RevitNodesTests/Elements/RevitLinkTests.cs [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+            var linkInstance = ElementSelector.ByElementId(linkInstanceId, true) as Revit.Elements.RevitLinkInstance;
+            
+            var wallsCategory = Revit.Elements.Category.ByName("Walls");
+            var wallsInLink = Revit.Elements.RevitLinkInstance.AllElementsOfCategory(linkInstance, wallsCategory);
+
+            List<object> wallGeometry = LinkElement.GetGeometry(wallsInLink.First(), "Medium") as List<object>;
+
+            Assert.NotNull(wallGeometry);
+            /*object t = wallGeometry.GetType();*/
```
Comment (dnenov) [inline test/Libraries/RevitNodesTests/Elements/RevitLinkTests.cs:323]: If we don't need the commented-out code, perhaps remove it?

**Comment by dnenov** [comment]
@Mikhinja - should we now merge this PR?

**Comment by dariaivanciucova** [comment]
hey @Mikhinja, is this still pending review?

**Comment by Mikhinja** [comment]
Sorry, I thought @dnenov had an additional change request, I'll merge it now

**Pull request merged**